src/components/questions/ImageUpload.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  ToastAndroid,
+  Alert,
+  PermissionsAndroid,
+} from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+const ImageUpload = () => {
+  const [image, SetImage] = React.useState('');
+
+  // Show toast alert messages
+  const setToastMsg = (msg: any) => {
+    ToastAndroid.showWithGravity(msg, ToastAndroid.SHORT, ToastAndroid.CENTER);
+  };
+
+  const uploadImage = () => {
+    launchImageLibrary({ mediaType: 'photo', quality: 1, includeBase64: true }, (response) => {
+      if (response.didCancel) {
+        setToastMsg('geen foto gekozen');
+      } else if (response.errorCode == 'permission') {
+        setToastMsg('permissie afgewezen');
+      } else if (response.errorCode == 'others') {
+        setToastMsg(response.errorMessage);
+      } else if (response.assets[0].fileSize > 2097152) {
+        Alert.alert('Maximum grootte verstreken', 'Selecteer een foto onder 2M B', [
```

Comment:
Verstreken zou ik aanpassen naar Overschreden

Reply:
M B > MB

Reply:
Vanwaar trouwens de cap op 2MB?

Reply:
Ik heb voor nu de cap op 2 mb gezet vanwege de hoeveelheid data dat dan gebruikt word voor het versturen van een foto. Maar dat kan nog altijd aangepast worden, ligt ook aan hoe de compressor volgend sprint gebruikt zal worden.

Reply:
Ja weet niet of je dat moet limiteren, als de gebruiker een foto heeft gemaakt en die is groter dan 2MB wilt die m waarschijnlijk gewoon uploaden ipv eerst zelf moeten resizen. Denk dat we er meer voor moeten zorgen dat de app zo min mogelijk data ophaalt/verstuurd wanneer niet nodig. De gebruiker kiest immers zelf de foto en upload deze

Reply:
Eens met Jaap. Is voor nu nog niet belangrijk.